src/Cairo.jl

```diff
@@ -311,10 +323,10 @@ end
 
 function write_to_png(surface::CairoSurface, filename::AbstractString)
     ccall((:cairo_surface_write_to_png,_jl_libcairo), Void,
-          (Ptr{UInt8},Ptr{UInt8}), surface.ptr, bytestring(filename))
+          (Ptr{UInt8},Ptr{UInt8}), surface.ptr, @compat(String(filename)))
 end
 
-writemime(io::IO, ::MIME"image/png", surface::CairoSurface) =
+@compat Base.show(io::IO, ::MIME"image/png", surface::CairoSurface) =
```

Comment:
I didn't realize you had it set up on your account---I thought it wasn't running PRs, because it didn't show up when I pushed things. I basically created a new account called JuliaGraphics, but it's emailing me. If you'd like to be the main administrator of this account, by all means please do so. I'll see if I can figure out how to set that up.